### PR TITLE
Add Tribute Labs DAO standard

### PR DESCRIPTION
**Pull request by benton-fried36**
Tribute Labs DAO standard

Added ITribute DAO standard
Added Tribute membership checker contract

Tests
Replaced SWLegacy contracts with Tribute contracts in `autID.test` and `daoExpander.test`. I can put the references to SWLegacy contracts back if needed.

### CODE REVIEW

File: test/autID.test.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ describe("AutID", function () {
       await daoTypes.deployed();
 
       const SWLegacyMembershipChecker = await ethers.getContractFactory(
-        "SWLegacyMembershipChecker"
+        "TributeMembershipChecker"
       );
```
Comment (migrenaa) [inline test/autID.test.js]: Can you please leave the SWLEgacyMembershipChecker here? 

Having tests on the TributeMembershipChecker is enough.

File: test/daoExpander.test.js [hunk 2]
```diff
@@ -153,7 +153,7 @@ describe("DAOExpander", function () {
   });
   describe("Manage URLs", async () => {
     before(async function () {
-      const Community = await ethers.getContractFactory("SWLegacyDAO");
+      const Community = await ethers.getContractFactory("Tribute");
```
Comment (migrenaa) [inline test/daoExpander.test.js]: Same here

**Review by migrenaa — CHANGES_REQUESTED** [review]

File: contracts/membershipCheckers/TributeMembershipChecker.sol [hunk 3]
```diff
@@ -0,0 +1,29 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "./IMembershipChecker.sol";
+import "../daoStandards/ITribute.sol";
+
+/// @title NewMembershipChecker
```
Comment (migrenaa) [inline contracts/membershipCheckers/TributeMembershipChecker.sol]: please edit the dev comments

File: contracts/membershipCheckers/TributeMembershipChecker.sol [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        view
+        override
+        returns (bool)
+    {
+        require(daoAddress != address(0), "AutID: daoAddress empty");
+        require(member != address(0), "AutID: member empty");
+
+        // implement your membership checker logic here
+        // return reasonable result ;)
+        // return false;
```
Comment (migrenaa) [inline contracts/membershipCheckers/TributeMembershipChecker.sol]: delete these comments

File: test/autID.test.js (hunk 1 above)
Comment (benton-fried36) [reply test/autID.test.js]: Done

**Review by migrenaa — APPROVED** [review]

File: contracts/membershipCheckers/TributeMembershipChecker.sol [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+pragma solidity ^0.8.0;
+
+import "./IMembershipChecker.sol";
+import "../daoStandards/ITribute.sol";
+
+/// @title NewMembershipChecker
+/// @notice Implementation of IMembershipChecker for your new DAO standard
+contract TributeMembershipChecker is IMembershipChecker {
+
+    /// @notice Implements a check if an address is a member of a DAO
```
Comment (migrenaa) [inline contracts/membershipCheckers/TributeMembershipChecker.sol:11]: relevant comment on the check

**Review by migrenaa — CHANGES_REQUESTED** [review]

**Review by migrenaa — APPROVED** [review]

**Pull request merged**